provision.sh

```diff
@@ -76,10 +76,10 @@ sudo modprobe -r kvm_intel
 sudo modprobe kvm_intel nested=1
 sudo systemctl restart libvirtd
 # Set up iptables and firewalld
```

Comment:
Can you please add some description to this commit, summarising what exactly is changing and why?

Reply:
@zeenix explained.

Reply:
Thanks. Makes perfect sense now. Some nitpicks that helps a slightly-dyslexic like me but no worry if you push without fixing them:

* `this purpose also to forward` -> `this purpose. Also to forward`
* `it is easily` -> `it is easy`
* `instead using` -> `instead of using`.

Many thanks!

Reply:
Done, merging now.